### PR TITLE
intf refactoring - logic, code conventions, var scope and styling

### PR DESCRIPTION
**Pull request by ishayhil**
Mutated vars access from global scope to local scope if possible. Some logic modifications and object initialization changes. Changed function names to lower case python style. Changed object names to upper case start names (like on lua convention doc).

### CODE REVIEW

File: intf/cnsr_intf.lua [hunk 1]
```diff
@@ -10,69 +11,72 @@ FRAME_INTERVAL = 30000
...
+-- mute
+MuteParams = {
+	start_time = 0,
+	end_time = 0,
+	activated = false,
+	prev_volume = vlc.volume.get(),
+	action_word = "muted"
+}
+
+function MuteParams.mute()
```
Comment (ophirhan) [inline intf/cnsr_intf.lua]: should be MuteParams.activate

File: intf/cnsr_intf.lua [hunk 2]
```diff
@@ -10,69 +11,72 @@ FRAME_INTERVAL = 30000
...
 
-hide_params.activate = hide
-
-
-
-function Looper()
-	next_loop_time = vlc.misc.mdate()
+-- Main app entry point
+function looper()
+	local next_loop_time
```
Comment (ophirhan) [inline intf/cnsr_intf.lua]: = vlc.misc.mdate()

File: intf/cnsr_intf.lua [hunk 3]
```diff
@@ -10,69 +11,72 @@ FRAME_INTERVAL = 30000
...
 		if vlc.playlist.status()~="stopped" and config.CNSR and config.CNSR.tags and #config.CNSR.tags ~= 0 then
 			tags = config.CNSR.tags
 
-			input = vlc.object.input()
+			local input = vlc.object.input()
 			current_time = vlc.var.get(input,"time")
 			check_disable_actions()
 			reverse = prev_time > current_time
-			tag = get_current_tag()
+			local tag = get_current_tag()
```
Comment (ophirhan) [inline intf/cnsr_intf.lua]: should receive "tags" since they are no longer global

File: intf/cnsr_intf.lua (hunk 1 above)
Comment (ishayhil) [reply intf/cnsr_intf.lua]: done

File: intf/cnsr_intf.lua (hunk 2 above)
Comment (ishayhil) [reply intf/cnsr_intf.lua]: done

File: intf/cnsr_intf.lua (hunk 3 above)
Comment (ishayhil) [reply intf/cnsr_intf.lua]: done

File: intf/cnsr_intf.lua [hunk 4]
```diff
@@ -98,28 +102,28 @@ function display_reason(reason_string, category, tag_end)
...
 
-	if hide_params.activated and (current_time > hide_params.end_time or current_time < hide_params.start_time) then
+	if HideParams.activated and (current_time > HideParams.end_time or current_time < HideParams.start_time) then
 		vlc.var.set(hide_filter, "video-filter", "")
-		hide_params.activated = false
+		HideParams.activated = false
 	end
 end
 
 function skip(skip_start, skip_end)
```
Comment (ophirhan) [inline intf/cnsr_intf.lua]: should receive input because it is no longer global

File: intf/cnsr_intf.lua [hunk 5]
```diff
@@ -146,23 +150,23 @@ function get_current_tag()
 end
 
 function check_collision()
-	if hide_params.activated and mute_params.activated then
-		local skip_end = math.min(mute_params.end_time, hide_params.end_time)
-		local skip_start = math.max(mute_params.start_time, hide_params.start_time)
+	if HideParams.activated and MuteParams.activated then
+		local skip_end = math.min(MuteParams.end_time, HideParams.end_time)
+		local skip_start = math.max(MuteParams.start_time, HideParams.start_time)
 		skip(skip_start, skip_end)
```
Comment (ophirhan) [inline intf/cnsr_intf.lua]: should send input

File: intf/cnsr_intf.lua [hunk 6]
```diff
@@ -146,23 +150,23 @@ function get_current_tag()
...
 	end
 end
 
 function execute_tag(tag)
-
+	local action_params
 	if tag.action == SKIP then
 		skip(tag.start_time, tag.end_time)
```
Comment (ophirhan) [inline intf/cnsr_intf.lua]: send input

File: intf/cnsr_intf.lua (hunk 4 above)
Comment (ishayhil) [reply intf/cnsr_intf.lua]: which input? I'm not sure I understand which var you refer to

Comment (ophirhan) [reply intf/cnsr_intf.lua]: skip uses input = vlc.object.input()

File: intf/cnsr_intf.lua (hunk 5 above)
Comment (ishayhil) [reply intf/cnsr_intf.lua]: done

File: intf/cnsr_intf.lua (hunk 6 above)
Comment (ishayhil) [reply intf/cnsr_intf.lua]: done

File: intf/cnsr_intf.lua (hunk 4 above)
Comment (ishayhil) [reply intf/cnsr_intf.lua]: right, but `check_disable_actions` doesn't use `skip` function

Comment (ophirhan) [reply intf/cnsr_intf.lua]: I'm referring to the skip function signature line 116

Comment (ishayhil) [reply intf/cnsr_intf.lua]: oh for some reason I see that you refer to line 112

**Pull request merged**